src/java.xml/share/classes/module-info.java

```diff
@@ -24,14 +24,93 @@
+ * on the command line and/or in the
+ * <a href="#ConfigurationFile">XML Library (java.xml) Configuration File</a>,
+ * without changing code in the later two options.
+ * <p>
+ * The {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}' RESOLVE
+ * property for example has defined a corresponding System Property
+ * {@code javax.xml.catalog.resolve} as listed in the table
+ * {@link javax.xml.catalog.CatalogFeatures Catalog Features} that gives it the
+ * flexibility to be set through the System API, or without making code changes,
+ * be initialized on the command line, or in the XML Library (java.xml)
+ * Configuration File as illustrated in the <a href="#PP">Property Precedence</a>
+ * section.
+ *
+ * <h2 id="ConfigurationFile">XML Library (java.xml) Configuration File</h2>
+ * JAXP supports the use of a configuration file for the
+ * <a href="#LookupMechanism">Factory Lookup Mechanism</a> and
+ * setting properties that have defined corresponding system properties.
+ *
+ * <h3>Format</h3>
+ * The configuration file must be in standard {@link java.util.Properties} format.
+ * <p>
+ * The keys are the names of the system properties, for example, those listed in
+ * column {@code System Property Name} of the table <a href="#Factories">JAXP Factories</a>,
+ * or {@code System Property} in the table {@code Catalog Features}
+ * of class {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}.
+ *
+ * <p>
+ * The values are those defined in the specific API or process, for example, the
+ * fully qualified name of the implementation class for the
+ * <a href="#LookupMechanism">Factory Lookup Mechanism</a>, or {@code value} in
+ * table {@code Catalog Features} of class
+ * {@link javax.xml.catalog.CatalogFeatures CatalogFeatures}.
+ *
+ * <p>
+ * Below are examples on what can be placed in the configuration file:
+ * <pre>
+ *     {@code javax.xml.parsers.DocumentBuilderFactory=packagename.DocumentBuilderFactoryImpl}
+ *     {@code javax.xml.catalog.resolve=strict}
+ * </pre>
+ *
+ *
+ * <h3 id="CF_Default">{@code jaxp.properties}</h3>
+ * By default, the <a href="#Factories">JAXP Factories</a> will look for a
+ * configuration file called {@code jaxp.properties} in the ${java.home}/conf
+ * directory and use the entries if any to customize the behavior of the XML
+ * factories and processors.
+ *
+ * <p>
+ * {@code jaxp.properties} will be read only once during the initialization of
+ * the JAXP implementation and cached in memory. If the file does not exist when
+ * the first attempt is made to read from it, no further attempts are made to check
+ * for its existence. It is not possible to change the value of any property after
+ * it has been read for the first time.
+ *
+ *
+ * <h3 id="CF_SP">User-defined Configuration File</h3>
+ * A system property {@systemProperty java.xml.config.file} can be set on the
+ * command line or through the System API (e.g. System.setProperty method) to
```

Comment:
"or through the System API (e.g. System.setProperty)" could be improved to make it clear that the system property can be set at run-time with System.setProperty (add link).